app/scripts/controllers/preferences.js

```diff
@@ -34,8 +32,7 @@ class PreferencesController {
       lostIdentities: {},
     }, opts.initState)
 
-    this.getFirstTimeInfo = opts.getFirstTimeInfo || null
-    this.notifier = opts.notifier || notifier
+    this.diagnostics = opts.diagnostics
```

Comment:
Should we default this to some fallback instance of the DiagnosticsReporter? Maybe this isn't necessary if we know it's always passed where this PreferencesController is instantiated.

Reply:
Never mind, I see this is defaulted in MetaMaskController.